src/main/xerus/monstercat/Settings.kt

```diff
@@ -42,6 +42,42 @@ object Settings : SettingsNode("xerus/monsterutilities") {
 	
 	val FILENAMEPATTERN = create("updatePattern", "MonsterUtilities-%version%.jar")
 	
+	enum class ConnectionSpeed(val maxConnections: Int) {
+		DIALUP(5) {
+			override fun toString() = "Dial-up (100 kb/s)"
```

Comment:
please pass that String in the constructor and return it in toString instead of overriding the method for each object individually.

Reply:
Independent toString : 4065a10c33d61e78f0da7914171930528d2d5efb